packages/liveblocks-react/src/factory.tsx

```diff
@@ -384,6 +384,26 @@ type RoomContextBundle<
     callback: F,
     deps?: unknown[]
   ): OmitFirstArg<F>;
+
+  suspense: {
+    useStorage(): ToImmutable<TStorage>;
+    useStorage<T>(
+      selector: (root: ToImmutable<TStorage>) => T,
+      isEqual?: (a: T, b: T) => boolean
+    ): T;
+
+    useSelf(): User<TPresence, TUserMeta>;
+    useSelf<T>(
+      selector: (me: User<TPresence, TUserMeta>) => T,
+      isEqual?: (a: T, b: T) => boolean
+    ): T;
+
+    useOthers(): Others<TPresence, TUserMeta>;
+    useOthers<T>(
+      selector: (others: Others<TPresence, TUserMeta>) => T,
+      isEqual?: (a: T, b: T) => boolean
+    ): T;
```

Comment:
Basically all the same selector functions, but without all the possible `null`s in the return types.